### PR TITLE
Simplified examples and added delayed connect

### PR DESCRIPTION
**Pull request by dpwspoon**

### CODE REVIEW

**Comment by dpwspoon** [comment]
Ping @jfallows @pkhanal, PTAL

File: driver/src/main/java/org/kaazing/k3po/driver/netty/bootstrap/http/HttpBootstrapFactorySpi.java [hunk 1]
```diff
@@ -77,4 +89,5 @@ public void shutdown() {
     public void releaseExternalResources() {
         // ignore
     }
+
 }
```
Comment (jfallows) [inline driver/src/main/java/org/kaazing/k3po/driver/netty/bootstrap/http/HttpBootstrapFactorySpi.java]: Are these changes still needed?

File: k3po-maven-plugin/pom.xml [hunk 2]
```diff
@@ -56,6 +56,41 @@
...
+        							<action>
+        								<ignore></ignore>
+        							</action>
+        						</pluginExecution>
+        					</pluginExecutions>
+        				</lifecycleMappingMetadata>
+        			</configuration>
+        		</plugin>
+        	</plugins>
+        </pluginManagement>
```
Comment (jfallows) [inline k3po-maven-plugin/pom.xml]: Smaller indentation?

File: lang/src/main/antlr4/org/kaazing/k3po/lang/parser/v2/Robot.g4 [hunk 3]
```diff
@@ -474,6 +474,10 @@ HttpVersionKeyword
     : 'version'
     ;
 
+WhenKeyword
+    : 'when'
+    ;
+
```
Comment (jfallows) [inline lang/src/main/antlr4/org/kaazing/k3po/lang/parser/v2/Robot.g4]: Remove?

File: lang/src/main/java/org/kaazing/k3po/lang/ast/AstConnectNode.java [hunk 4]
```diff
@@ -77,6 +78,18 @@ protected boolean equalTo(AstConnectNode that) {
     @Override
     protected void describeLine(StringBuilder sb) {
         super.describeLine(sb);
-        sb.append(String.format("connect %s\n", location));
+        if (barrier == null) {
+            sb.append(String.format("connect %s\n", location));
+        } else {
+            sb.append(String.format("connect await %s\nconnect %s\n", barrier, location));
```
Comment (jfallows) [inline lang/src/main/java/org/kaazing/k3po/lang/ast/AstConnectNode.java]: Static imports for String.format for readability?

File: specification/sse/pom.xml [hunk 5]
```diff
@@ -15,14 +15,18 @@
...
     </scm>
 
-
     <dependencies>
+		<dependency>
+			<!-- Needed for EL only, not AST -->
+			<groupId>org.kaazing</groupId>
+			<artifactId>k3po.lang</artifactId>
+			<version>${project.version}</version>
+		</dependency>
```
Comment (jfallows) [inline specification/sse/pom.xml]: Smaller indentation?

File: specification/sse/src/main/java/org/kaazing/specification/sse/internal/Functions.java [hunk 6]
```diff
@@ -19,16 +19,30 @@
...
+		public Mapper() {
+			super(Functions.class);
+		}
 
-    // TODO
-    @Override
-    protected void cleanUp() {
+		@Override
+		public String getPrefixName() {
+			return "wse";
```
Comment (jfallows) [inline specification/sse/src/main/java/org/kaazing/specification/sse/internal/Functions.java]: "sse" ?

File: driver/src/test/java/org/kaazing/k3po/driver/BehaviorIT.java [hunk 7]
```diff
@@ -42,9 +42,23 @@
 
     @Test
     @TestSpecification({
-        "many.expressions" })
+        "many.juel.expressions" })
```
Comment (jfallows) [inline driver/src/test/java/org/kaazing/k3po/driver/BehaviorIT.java]: "many.el.expressions" ?  (JUEL is implementation of EL, syntax in scripts is EL)

**Comment by jfallows** [comment]
Looks great, @dpwspoon !

File: driver/src/main/java/org/kaazing/k3po/driver/netty/bootstrap/http/HttpBootstrapFactorySpi.java (hunk 1 above)
Comment (dpwspoon) [inline driver/src/main/java/org/kaazing/k3po/driver/netty/bootstrap/http/HttpBootstrapFactorySpi.java]: Fixed

**Comment by dpwspoon** [comment]
@jfallows Thanks! Feedback applied.
